src/main/resources/db/migration/local/afterMigrate.sql

```diff
@@ -23,3 +23,7 @@ SELECT -1, sg.grp_id FROM darts.security_group sg
 where sg.grp_id not in (select sgua.grp_id from darts.security_group_user_account_ae sgua where sgua.usr_id = -1)
 ORDER BY 2;
 
+INSERT INTO darts.security_group_courthouse_ae (grp_id, cth_id)
```

Comment:
this will add courthouse 1 to all security groups, is it not that we want a particular security group to be able to be a super-user and have permissions to all courthouses, not just 1?